docs/_future/triggers.md

```diff
@@ -0,0 +1,138 @@
+
+The `create` method for runtime Triggers takes the form of `client.workflows.triggers.create()`. Within that method you can leverage `inputs` acquired in previous steps (i.e. with Functions) within the Workflow.
+
+```js
+const triggerResponse = await client.workflows.triggers.create({
+  type: "shortcut",
+  name: "My Trigger",
+  workflow: "#/workflows/my_workflow",
+  inputs: {
+    input_name: {
+      value: "value",
+    }
+  }
+});
+```
+
+The response will have a property called `ok`. If `true`, then the Trigger was created, and the `trigger` property will be populated.
+
+Your response will include a `trigger.id`; be sure to store it! You use that to `update` or `delete` the Trigger if need be.
+
+#### Development and production Triggers
+
+The Triggers you create when you're running your app locally in a development environment (with the `slack run` command) will not work when you [deploy your app](/bolt-js/future/deploy-your-app). You'll need to `create` any Triggers again with the CLI.
+
+---
+
+### Update a Trigger {#update}
+
+Make an update to a pre-existing Trigger by using the `slack trigger update` command. Provide the same payload you used to create the trigger *in its entirety*, in addition to the Trigger ID.
+
+```bash
+slack trigger update --trigger-id Ft123ABC --trigger-def "path/to/trigger.json"
+```
+---
+
+### Delete a Trigger {#delete}
+
+You can delete a Trigger with the `slack trigger delete` command.
+
+```bash
+slack trigger delete --trigger-id FtABC123
+```
+---
+
+### Trigger types {#types}
+
+There are four types of Triggers, each one having its own specific implementation. To learn more about each trigger, visit the linked pages below.
+
+| Trigger Type                     | Use case                                                      |
+|----------------------------------|---------------------------------------------------------------|
+| [Link Triggers](https://api.slack.com/future/triggers/link)           | Invoke a Workflow from anywhere in Slack                      |
+| [Scheduled Triggers](https://api.slack.com/future/triggers/scheduled) | Invoke a Workflow at specific time intervals                  |
+| [Event Triggers](https://api.slack.com/future/triggers/event)         | Invoke a Workflow when a specific event happens in Slack      |
+| [Webhook Triggers](#https://api.slack.com/future/triggers/webhook)     | Invoke a Workflow when a specific URL receives a POST request |
+
+---
+
+### Manage access {#manage-access}
+
+A newly created Trigger will only be accessible to others inside a workspace once its creator has granted access.
+
+To manage what user (or many users) have access to run your Triggers, you'll use the `access` command.
+
+To learn more about this, visit the guide [here](https://api.slack.com/future/triggers#manage-access).
+
+---
+
+### Conclusion {#conclusion}
```

Comment:
note to self: maybe from here we can direct them to the deploy guide

Reply:
(additional note - we'll add that in once the deploy guide is ready! resolving this comment)